scripts/travis_ubuntu.sh

```diff
@@ -3,10 +3,17 @@
 ## -x be verbose; write what we are doing, as we do it
 set -ex
 
+cmd=ls
+if [ $3 = "old" ]; then
+    cmd="sed -i 's/\(security\|archive\).ubuntu/old-releases.ubuntu/g' /etc/apt/sources.list"
+fi 
+
```

Comment:
> If this is removed : build fails ?

Yes. Because `http://archive.ubuntu.com/ubuntu/dists/eoan*` no longer exists, you won't be able to install anything with the original `sources.list`.